### PR TITLE
Replaces approx 280 potentially offensive words from word list

### PR DESCRIPTION
**Pull request by sts10**
I filtered out a list of "bad words" I found a while ago and replaced them with common words from [Google Ngram data](https://github.com/sts10/common_word_list_maker). I did this operation using a tool I made called [Tidy](https://github.com/sts10/tidy).

Obviously this is a sensitive and subjective area, and some profane words may be left and some innocent words may have been removed, but I think overall this is an improvement on the current list.

### CODE REVIEW

File: lib/wordlist.js [hunk 1]
```diff
@@ -14683,7 +14755,6 @@ module.exports = [
 'dilative',
 'dilatorily',
 'dilatory',
-'dildo',
 'dildoe',
```
Comment (diracdeltas) [inline lib/wordlist.js]: could you maybe replace this too while you're at it? thanks

Comment (sts10) [reply lib/wordlist.js]: Done!

**Review by diracdeltas — CHANGES_REQUESTED** [review]
thanks for doing this - is there an easy way for you to replace these words without re-adding plurals? i ask because i originally filtered plurals out of the initial list to avoid confusion with the singular forms.

personally i would prefer to just replace the overtly offensive words (`dark**`, etc.) versus less-controversial ones like `ejaculate` (esp since the other forms of that word are still on the list) and `european` / `ethnic`. also i'm not sure why `jew` was taken off and `jews` was added for instance.

**Comment by sts10** [comment]
> is there an easy way for you to replace these words without re-adding plurals?

Not with my tool currently, no, but for this latest attempt, I used vim to remove words ending in "s" and "ed" and went from there. 

I'm curious about your worry about plurals though... pretty low odds that a user gets the singular and plural form of the same word in a given passphrase. 

> personally i would prefer to just replace the overtly offensive words (dark**, etc.) versus less-controversial ones like ejaculate (esp since the other forms of that word are still on the list) and european / ethnic. also i'm not sure why jew was taken off and jews was added for instance.

Agreed. I'm working with a 2,800-word list of profane words I gathered from multiple sources, so it's not very consistent with (a) how profane is too profane and (b) covering all conjugations/plurals. Kind of a work in progress.

Just made a new set of (397) replacements. Hopefully fits to these notes!

**Comment by sts10** [comment]
Hold on, trying again.

**Comment by sts10** [comment]
Phew, OK, down to 278 words replaced. I tried to target only the worst of 'em. 

I'll admit that the procedure I'm using to make changes is a bit convoluted, so mistakes may have been made. If my commits/edits aren't working out, [here is a list of "common" words that I think are not on your current list, and thus could be used to replace problematic words](https://gist.github.com/sts10/4f436eb0422171b5b8d0e5590ba3b035).

**Comment by diracdeltas** [comment]
thank you! i will review this tonight

**Comment by diracdeltas** [comment]
cleaned it up a bit more manually. i will open this as a new PR with some test fixes

**Comment by diracdeltas** [comment]
closing in favor of https://github.com/diracdeltas/niceware/pull/53

**Pull request closed without merging**